.github/workflows/build.yml

```diff
@@ -29,10 +29,21 @@ jobs:
       - name: Homebrew install dependencies
         # Compared to the README, adds ccache for faster compilation times
         # Compared to the emulator, adds p7zip.
-        run: >
-          brew install -q nasm ragel binutils coreutils libtool autoconf automake cmake makedepend
-          sdl2 lua@5.1 luarocks gettext pkg-config wget gnu-getopt grep bison
-          ccache p7zip
+        run: |
+          packages=(
+            nasm binutils coreutils libtool autoconf automake cmake makedepend
+            sdl2 lua@5.1 luarocks gettext pkg-config wget gnu-getopt grep bison
+            ccache p7zip
+          )
+          # Lua 5.1 is disabled, so we need to work around that:
+          # - fetch all packages
+          brew fetch "${packages[@]}"
+          # - disable auto-updates
+          export HOMEBREW_NO_AUTO_UPDATE=1
+          # - install lua@5.1 from cache
+          brew install "$(brew --cache lua@5.1)"
```

Comment:
Huh, this is easier than I'd expected.

Reply:
I was surprised too.

Reply:
Let's hope they don't cull those old bottles. 🤞